include/api/CHierarchicalResultsWriter.h

```diff
@@ -153,6 +153,8 @@ class API_EXPORT CHierarchicalResultsWriter : public model::CHierarchicalResults
                                const TResultWriterFunc& resultWriter,
                                const TPivotWriterFunc& pivotsWriterFunc);
 
+    virtual ~CHierarchicalResultsWriter() override = default;
```

Comment:
For all other overridden virtual functions we're not specifying virtual as well, so:

```suggestion
   ~CHierarchicalResultsWriter() override = default;
```